src/main/resources/application.yml

```
@@ -94,4 +94,13 @@ cloud:
-      static: ${AWS_REGION}
\ No newline at end of file
+      static: ${AWS_REGION}
+
+# 모니터링 관련 설정
+management:
+  server:
+    port: 9090
+  endpoints:
+    web:
+      exposure:
+        include: health,prometheus
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)